src/pudl/metadata/fields.py

```diff
@@ -245,6 +245,41 @@
         # TODO: Disambiguate if necessary. Does this mean different things in
         # different tables? It shows up in a lot of places.
     },
+    "capex_annual_addition": {
+        "type": "number",
+        "description": "Annual capital addition into `capex_total`.",
+        "unit": "USD",
+    },
+    "capex_annual_addition_rolling": {
+        "type": "number",
+        "description": "Year-to-date capital addition into `capex_total`.",
+        "unit": "USD",
+    },
+    "capex_annual_per_kw": {
+        "type": "number",
+        "description": "Annual capital addition into `capex_total` per kw.",
```

Comment:
It's a tad inconsistent to have `capex_annual_addition` and `capex_annual_XX` refer to the same type of thing. Maybe we could change `capex_annual_addition` to just `capex_annual`?

Reply:
Moved to #2533